### ISSUE TITLE
🚀 Feature: OpenTelemetry Metrics

### ISSUE THREAD

**Issue by nirga** [traceloop/openllmetry-js#185]
### Which component is this feature for?

All Packages

### 🔖 Feature description

Some values, like token usage, are better reported (also) as [otel metrics](https://opentelemetry.io/docs/specs/otel/metrics/). We should update all instrumentations to send metrics as well as traces.

### 🎤 Why is this feature needed ?

More ways to use the outputted data. Observability platforms can use metrics to set alerts on or to create dashboards.

### ✌️ How do you aim to achieve this?
Similar to the issue in OpenLLMetry - https://github.com/traceloop/openllmetry/issues/251
This needs to be researched and defined. Please consult on [slack](traceloop.com/slack) before starting to work on this.

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### Are you willing to submit PR?

None

**Comment by detunjiSamuel** [comment]
Hey, I would love to work on this issue.

**Comment by nirga** [comment]
Sounds good @detunjiSamuel! You can look at the python project we've already done some work there in the SDK and the OpenAI instrumentation

**Comment by deepshahcode** [comment]
I would  be glad to work on this issue.

**Comment by nirga** [comment]
Amazing @deepshahcode! Let me know if you need any help
